files/faban/src/cloud/benchflow/collectors/faban.go

```diff
@@ -67,9 +67,9 @@ func backupHandler(w http.ResponseWriter, r *http.Request) {
 	if err != nil {
 		panic(err)
 		} 
-	minioKey := minio.GenerateKey(path)
+	minioKey := minio.GenerateKey("faban")
```

Comment:
Right that's why I think we should have the same has for all the faban files belonging to the same trial, hence the use of the `fabanPath` variable that in your case should be the `faban trial id` send from the client. So we have `faban` in the name of the path, that is the name of the collector, and then we find all the data of the same trial in the same `hash` since we access then in this way.